src/pages/Login.tsx

```diff
@@ -9,16 +9,18 @@ export const Login = () => {
           padding: '0 !important'
         }}
       >
-        <Stack height="100vh" flexDirection="row" pr="19px">
-          <img
-            src="/static/login/image-login.png"
-            style={{
-              height: '100%',
-              width: '100%',
-              objectFit: 'cover'
-            }}
-          />
-          <Box py="18px" maxWidth="578px">
+        <Stack height="100vh" direction="row" py="18px">
+          <Box height="100%">
+            <img
+              src="/static/login/image-login.png"
+              style={{
+                height: '100%',
+                width: '100%',
+                objectFit: 'cover'
+              }}
+            />
+          </Box>
+          <Box maxWidth="578px" height="100%" mr="19px">
```

Comment:
```suggestion
          <Box minWdith="578px" height="100%" mr="19px">
```
harusnya disini minWidth atau maxWidth ya amannya?

Reply:
udah aman harusnya kak, soalnya yang img nya di lapis sama box, jadi box formsection nya itu width nya segitu, box image nya sesuaiin dengan sisa nya. 
kalo yang sebelumnya karna ga dilapisin sama box, width dari image nya itu selalu 100%

Reply:
kalo pake minwidth pas waktu tablet terlalu gede formSection nya.

Reply:
minWidth :
![image](https://github.com/PurwadhikaDev/purwadhika-corporate-website/assets/82017999/0a3e79a2-9235-4dfc-8dcb-e39c390c6fe2)

Reply:
sipp noted